app/design/frontend/Magento/luma/web/css/source/_extends.less

```diff
@@ -1913,6 +1913,36 @@
             display: table-cell;
         }
     }
+
+    .abs-discount-code {
```

Comment:
@abrarpathan19 can you please move the fix to Magento_Checkout module of luma theme